core/decoders/zrle.js

```diff
@@ -0,0 +1,246 @@
+/*
+ * noVNC: HTML5 VNC client
+ * Copyright (C) 2019 The noVNC Authors
+ * (c) 2012 Michael Tinglof, Joe Balaz, Les Piech (Mercuri.ca)
```

Comment:
This doesn't look correct. I guess you copied this header from some other file?

Reply:
I did, and I fixed it too now :)